vlayer/ethereum/circuit/lib/src/slot_test.nr

```diff
@@ -0,0 +1,39 @@
+mod test_mapping {
+    use crate::slot::mapping;
+
+    #[test]
+    fn adress_mapping() {
+        let slot = [
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 9
+        ];
+        let address = [
+            0x55, 0xfe, 0x00, 0x2a, 0xef, 0xf0, 0x2f, 0x77, 0x36, 0x4d, 0xe3, 0x39, 0xa1, 0x29, 0x29, 0x23, 0xa1, 0x58, 0x44, 0xb8
+        ];
+        assert_eq(
+            [
+            0x57, 0xd1, 0x8a, 0xf7, 0x93, 0xd7, 0x30, 0x0c, 0x4b, 0xa4, 0x6d, 0x19, 0x2e, 0xc7, 0xaa, 0x09, 0x50, 0x70, 0xdd, 0xe6, 0xc5, 0x2c, 0x68, 0x7c, 0x6d, 0x0d, 0x92, 0xfb, 0x85, 0x32, 0xb3, 0x05
+        ], mapping(slot, address)
+        );
+    }
+
+    #[test]
+    fn mapping_of_mapping() {
```

Comment:
This test data comes from https://dashboard.tenderly.co/tx/mainnet/0xa93ce4ab1e8b39272a6f385694281d8381901fd5c7fa45499c82a98188780e92/state-diff

<img width="960" alt="image" src="https://github.com/vlayer-xyz/noir-ethereum-history-api/assets/2027578/e6723311-717f-4c45-b438-847c3523b89d">

Reply:
👏